elements/storytelling/src/main.js

```diff
@@ -67,8 +84,20 @@ export class EOxStoryTelling extends LitElement {
     // Check if 'markdown' property itself has changed and generate sanitized html
     if (changedProperties.has("markdown")) {
       this.#html = DOMPurify.sanitize(md.render(this.markdown));
+      if (this.enableNav) this.nav = md.nav;
       this.requestUpdate();
     }
+
+    if (changedProperties.has("nav")) {
+      this.shadowRoot.querySelectorAll(".navigation a").forEach((doc) => {
+        doc.addEventListener("click", (e) => {
+          e.preventDefault();
```

Comment:
The `preventDefault` here is needed inside the storybook iframe so the page doesn't navigate. But in a "normal" environment, the default is actually nice because it adds the hash with the id to the url, so one could link to a specific section of the story by copying the url.

Maybe we can prevent the default but still manually add the hash to the url instead. When loading the url with the has, it'd be nice if it would directly scroll to the correct section.

Reply:
Done ✅